tutorial/source/intro_part_ii.ipynb

```diff
@@ -141,30 +115,9 @@
xAgMHz4c8fHxckQgIhKW2Yu/qqoKixYtwpYtW5CUlISvv/4aN2/eNHcMIiJhmb34\nMzIy0LFjR3To0AHW1tYICgpCamqquWMQEQnL7Nfx5+XlwdXVVf+9i4sLMjIy6p1HrbY1eLzDn2sM\nnlcuL7O+gGWuMxHV7WU74Vk8uUtEJBizF7+Liwvu37+v/z4vLw8uLi7mjkFEJCyzF3+vXr1w+/Zt\n3LlzB+Xl5UhKSoK/P29tQERkLmY/xq9SqfDRRx9h2rRpqKqqwujRo9G1a1dzxyAiEpZCkiRJ7hBE\nRGQ+PLlLRCQYFj8RkWCadPH7+/sjJCQEGo0GYWFhcsep108//YTo6GiMHDkSgYGBuHTpktyR6nTr\n1i1oNBr9v759+2Lbtm1yx6rTtm3bEBQUhODgYMTExKCsrEzuSPXavn07goODERQU1Oi267x58+Dj\n44Pg4GD9tAcPHiAiIgIBAQGIiIjAw4cPZUxYXW15jx49iqCgIHTv3h1XrlyRMV11tWVdsWIFRo4c\niZCQEERFReGnn34yzmBSEzZkyBBJp9PJHaNB5s6dK+3bt0+SJEkqKyuTHj58KHOihqmsrJQGDhwo\n5eTkyB2lVvfv35eGDBkilZaWSpIkSdHR0dKBAwdkTlW3GzduSEFBQVJJSYlUUVEhTZ48Wbp9+7bc\nsfTS09Ol7777TgoKCtJPW7FihbRp0yZJkiRp06ZN0meffSZXvBpqy3vz5k0pMzNTeuutt6SMjAwZ\n01VXW9ZTp05JFRUVkiRJ0meffWa0bduk9/gtxaNHj3Du3Dn8/ve/BwBYW1vDzs5O5lQNk5aWhg4d\nOsDNzU3uKHWqqqrC48ePUVlZicePH8PZ2VnuSHXKzMzEa6+9hpYtW0KlUsHb2xvJyclyx9Lz9vaG\nvb19tWmpqakIDQ0FAISGhiIlJUWOaLWqLW+XLl3g7u4uU6K61ZbV19cXKtXTiy/79OlT7TNQL6PJ\nF39ERATCwsKwd+9euaPUKScnB46Ojpg3bx5CQ0OxYMEClJSUyB2rQZKSkqr9atrYuLi4IDIyEkOG\nDIGvry9sbGzg6+srd6w6devWDRcuXEBhYSFKS0tx8uRJo/1nNxWdTqd/M1Wr1dDpdDInapoOHDiA\nQYMGGWVZTbr4d+/ejUOHDmHz5s3YtWsXzp07J3ekWlVWVuLq1asIDw9HYmIiWrZsaRG3qy4vL8c3\n33yDkSNHyh2lTg8fPkRqaipSU1Nx6tQplJaW4tAh+f4O8/N06dIF06ZNw9SpUzFt2jR0794dVlaW\n899UoVBAoVDIHaPJ2bBhA5RKJUaNGmWU5VnOK8oAP98KwsnJCcOHD3/uzeDk4urqCldXV/Tu3RsA\nMHLkSFy9elXmVM938uRJ9OzZE23btpU7Sp3OnDmD9u3bw9HREc2aNUNAQECjPnEOAGPGjEFCQgJ2\n7doFe3t7dOrUSe5I9XJycoJWqwUAaLVaODo6ypyoaUlISMDx48excuVKo72pNtniLykpQVFRkf7r\n06dPN9pPCKvVari6uuLWrVsAnh4379Kli8ypni8pKQlBQUFyx6hXu3btcPnyZZSWlkKSJIvYtj8f\nKrl37x6Sk5MREhIic6L6+fv7IzExEQCQmJiIoUOHypyo6Th58iS2bNmCDRs2oGXLlkZbbpP95O6d\nO3cQFRUF4OnJveDgYPzhD3+QOVXdrl27hgULFqCiogIdOnTAsmXLapzoaUxKSkowZMgQpKSkwNbW\nuLeMNba1a9fiyJEjUKlU6NGjB5YsWQJra2u5Y9VpwoQJePDgAVQqlf4Sv8YiJiYG6enpKCwshJOT\nE959910MGzYMs2fPRm5uLtq1a4c1a9agTZs2ckcFUHveNm3aYPHixSgoKICdnR169OiBrVu3yh21\n1qzx8fEoLy/Xb8/evXtj0aJFLz1Wky1+IiKqXZM91ENERLVj8RMRCYbFT0QkGBY/EZFgWPxERIJh\n8RO9oLy8PEycOLFBz/Xw8EBxcXGtj8XFxaG8vNyY0YgahJdzEpmQh4cHLl68iNatW7/QY0SmxD1+\nEtKePXuwcOFCAEBGRgY8PDz0t/T45JNPsHfvXly+fBkTJ05EWFgYwsLCcPz4cQBPb6rXv39//bKO\nHTuGkSNHIjQ0FBs3bqyxl79jxw6MHj0aQ4cOxbFjxwBAP/b48eOh0WiMd591ooYwys2diSzM7du3\npREjRkiSJEkbN26Uxo0bp7+nfEBAgHTlyhVJo9FIeXl5kiRJUl5enuTn5yc9fPhQunPnjtSvXz9J\nkiQpPz9f6tevn5SVlSVJkiR9+eWXUrdu3aSioiJJkiSpW7du0o4dOyRJkqTz589Lvr6++gy/fB6R\nOXGPn4TUsWNHlJWV4f79+0hLS8OcOXOQlpaG3NxcVFRUQKfTIScnB9OnT4dGo8H06dOhUCiQnZ1d\nbTmXL1/Gb37zG/2N1EaPHl1jrDfffBPA0/upa7XaRv8XwKjpU8kdgEguAwYMwLfffgudTof+/ftj\n8eLFOH78OPr37w9JkuDh4YFdu3bVmC8nJ+eFxmnevDkAQKlUAnh6G+6fpxHJgXv8JKwBAwZg8+bN\n8PT0BAD07dsXmzdvho+PDzw9PZGdnY1///vf+udnZGRAeuZaiN69e+Pq1av48ccfAQAHDx5s8Pit\nW7fW30GWyJxY/CSsAQMG4O7du/q7X/78/YABA2Bvb4+//vWvWL9+PUaNGoXAwECsW7euRvG3bdsW\nn3zyCaZPn47Q0FAUFBSgWbNmDbqFbmRkJCZNmsSTu2R2vJyT6CUVFRXBxsYGwNM/j7d//37s3r1b\n5lREdeMxfqKXtGPHDvzzn/9EVVUV7O3t8emnn8odiahe3OMnIhIMj/ETEQmGxU9EJBgWPxGRYFj8\nRESCYfETEQnm/wEMxSk3cXS/lQAAAABJRU5ErkJggg==\n",
```

Comment:
This is removing the image. Is this intentional?

Reply:
can someone make sure the diagrams in there match [the tutorial](http://pyro.ai/examples/intro_part_ii.html) ? i cant seem to pull this branch locally